Functions/ImageProcessing/xASL_im_ASLSubtractionAveraging.m

```diff
@@ -207,38 +212,37 @@
     [Fpath, Ffile] = xASL_fileparts(path_ASL4D);
     sideCarMat = fullfile(Fpath, [Ffile '.mat']);
 
-    if exist(sideCarMat, 'file')
+	if exist(sideCarMat, 'file') && nVolumes>1
```

Comment:
This we would need to check then. There should never be a sidecar .mat with different dimensions than the NIfTI, our pipeline and SPM ensures this. So I'm fine with this addition but then we also need a warning for the case where the sideCarMat exists and nVolume==1 because then something illegal happened in our pipeline ;)

Reply:
I added the nVolumes>1 ? I thought you added this :) That would be funny indeed.
I think I fixed the ASL4D loading, but I will check your fix.

Reply:
Both conditions were already there in two consecutive IFs. I've just merged them to a single IF. So the functionality remained unchanged. The only reason I changed this was that otherwise I would have to add another 'else'.

So the code stayed exactly as it was. You don't have to check anything.